package.json

```
@@ -35,6 +35,7 @@
     "babel-loader": "^8.2.3",
     "babel-polyfill": "^6.26.0",
     "dayjs": "^1.11.5",
+    "eslint-config-fashiondata": "git+https://github.com/FashionData/eslint-config-fashiondata",
```

J'aurais juste rename le nom de la dépendance et le repo :

```suggestion
    "@fashiondata/eslint-config": "git+https://github.com/FashionData/eslint-config",
```

---

Done